project_event/models/project_project.py

```diff
@@ -123,8 +123,18 @@ def get_message_body(self, action):
         return switcher.get(action)
 
     def get_message(self, action):
```

Comment:
@eilst Same method name exists in the project-task model. They are not related but do the same job. Better to change unique name specific to the model